komodo/local.py

```diff
@@ -0,0 +1,11 @@
+import operator
+from jinja2 import Template
+
+
+def write_local_activators(data, defs, local, local_csh):
```

Comment:
We need to to use this as well, right? could not find any usage except in test..

Reply:
The call is `local.write_local_activators` and that is used around line 70 in `cli.py`